test/users.test.js

```diff
@@ -119,3 +119,32 @@ describe('Post Sign In User', () => {
     done();
   });
 });
+
+describe('Get Users', () => {
+  test('Should fail for unauthenticate user', async done => {
+    const response = await request(app)
+      .get('/users')
+      .send();
+    expect(response.text).toContain('Please sign in');
+    expect(response.status).toBe(401);
+    done();
+  });
+
+  test('Should work for authenticate user', async done => {
+    await request(app)
+      .post('/users')
+      .send(user.dataValues);
+    const { body } = await request(app)
+      .post('/users/sessions')
+      .send({
+        email: user.dataValues.email,
+        password: 'contrasena1234'
+      });
```

Comment:
Since this is not what you are testing and maybe its going to be used in every tests form now on, find a way to implicitly execute this